build.gradle

```diff
@@ -1,21 +1,16 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.appcompat_version = '27.1.1'
```

Comment:
Why moving this versions? I have always seen it in the 'global' gradle scope

Reply:
It's been always weird for me to have these versions in a file where are never used while they are only used in another file